mlir/test/Quantum/ConversionTest.mlir

```diff
@@ -367,12 +367,12 @@ func.func @hamiltonian(%obs : !quantum.obs, %p1 : memref<1xf64>, %p2 : memref<3x
 // Measurements //
 //////////////////
 
-// CHECK: llvm.func @__quantum__qis__Measure(!llvm.ptr<struct<"Qubit", opaque>>) -> !llvm.ptr<struct<"Result", opaque>>
+// CHECK: llvm.func @__quantum__qis__Measure(!llvm.ptr<struct<"Qubit", opaque>>, i8) -> !llvm.ptr<struct<"Result", opaque>>
 
 // CHECK-LABEL: @measure
 func.func @measure(%q : !quantum.bit) -> !quantum.bit {
 
-    // CHECK: llvm.call @__quantum__qis__Measure(%arg0)
+    // CHECK: llvm.call @__quantum__qis__Measure(%arg0, %0)
```

Comment:
It is not a good idea to check for specific variable names like `%0`. (Yes, `%arg0` is kind of specific as well). You can keep track of the operations and variables with a syntax by FileCheck. E.g.,

```llvmir
[[variable:%.+]] = arith.constant 0.0{{.+}}+00 : f64
other.operation [[variable]]
```

It uses pattern matching. Anything between `[[` and `]]` is an a variable name assignment. The variable name is followed by a regex of what you want to match. So in this case, the string that matches the regex `%.+` will be assigned to `variable`. You can then use `[[variable]]` to refer to that string. [See here for more documentation](https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-regex-matching-syntax)


I think another test here where you add postselect to measure would also be a good idea. E.g.

```
%res, %new_q = quantum.measure %q %postselectvalue : i1, !quantum.bit
```

Reply:
Done